docs/workflows.rst

```diff
@@ -0,0 +1,153 @@
+================
+GitHub Workflows
+================
+
+All Python documentation translation repositories are hosted on `GitHub <https://github.com>`_.
+GitHub provides a very useful `CI <https://en.wikipedia.org/wiki/Continuous_integration>`_
+system, *GitHub Actions*, which runs *workflows*. Workflows are particularly useful
+for testing projects, in translation repos they can be used to update translations
+from Transifex, lint translated files, and test build the translated documentation.
+
+This repository provides three sample workflows, a Linting workflow, a Test Build
+workflow, and a workflow for pulling translations from Transifex. They can be found
+in the `sample-workflows <https://github.com/python-docs-translations/transifex-automations/tree/main/sample-workflows>`_
+and explanations of how they work and what you need to do to set them up can be found below.
+
+.. seealso::
+   - `GitHub docs: Writing workflows <https://docs.github.com/en/actions/writing-workflows>`_
+   - :doc:`Setting up Transifex <commands.rst>`
+
+Running Workflows
+-----------------
+
+To run on of our provided workflows (or any workflow), put the file in the
+``.github/workflows`` folder and it will run automatically depending on how it
+is configured.
+
+General Overview
+----------------
+
+.. code-block:: workflow
```

Comment:
Considering that "workflow" is not a valid lexical parser name, I'm suggesting to use "yaml" since these workflows are written in YAML. This will fix the syntax highlighting.

```suggestion
.. code-block:: yaml
```